modules/python/init.zsh

```diff
@@ -92,22 +92,31 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) && \
   # Disable the virtualenv prompt.
   VIRTUAL_ENV_DISABLE_PROMPT=1
 
-  # Enable 'virtualenv' with 'pyenv'.
-  if (( $+commands[pyenv] )) && \
-     pyenv commands | command grep -q virtualenv-init
-  then
-    eval "$(pyenv virtualenv-init -)"
-    # Optionall activate 'virtualenvwrapper' with 'pyenv' is available.
-    if (( $#commands[(i)pyenv-virtualenvwrapper(_lazy|)] )); then
-      pyenv "${${(@O)commands[(I)pyenv-virtualenvwrapper(_lazy|)]}[1]#pyenv-}"
+  if (( $+commands[pyenv] )); then
```

Comment:
I think this might be a regression for @diraol, since this might revert the changes in #1414